VsViewer/vs_calc/spt_vs_correlations.py

```diff
@@ -56,16 +102,16 @@ def effective_stress_brandenberg(
         else:
             sigma = 0.2
         return stress, sigma, tao, b0, b1, b2
-    elif soiltype.value == SoilType.Silt.value:
+    elif soiltype == SoilType.Silt:
         b0 = 3.783
         b1 = 0.178
         b2 = 0.231
         tao = 0.227
         # (Brandendberg et al, 2010)
         if depth > water_table_depth:
-            stress = water_table_depth * 19 + (depth - water_table_depth) * (17 - 9.81)
+            stress = water_table_depth * 17 + (depth - water_table_depth) * (19 - 9.81)
```

Comment:
You've implemented all of the changes for this file correctly in terms of what I had and that's good for now. One comment I'll make is the way the effective stress is calculated here isn't right but we will fix that after this PR and it may require some additional functionality.

Reply:
Good to know

Reply:
@joelridden maybe add a TODO to that line?

Reply:
Good idea, I'll add TODO to both this and the stdev issue with the spt correlation